app/controllers/organizations/page_texts_controller.rb

```diff
@@ -8,14 +8,14 @@ def update
     if @page_text.update(page_text_params)
       redirect_to edit_page_text_path, notice: "Page text updated successfully!"
     else
-      render :edit, status: :unprocessable_entity
+      redirect_to edit_page_text_path, alert: @page_text.errors.full_messages.to_sentence
     end
   end
 
   private
 
   def page_text_params
-    params.require(:page_text).permit(:hero, :about)
+    params.require(:page_text).permit(:hero, :about, :hero_image, about_us_images: [])
   end
 
   def set_page_text
```

Comment:
This is slightly unrelated but I found a bug while testing - could you please change line 22 from 
`@page_text = current_user.organization.page_text` to
`@page_text = PageText.first` ?
I tried changing the slug from Baja to Alta and if gives the following error because the user context doees not exist after redirecting to the Alta site 
```
undefined method `organization' for nil:NilClass
    @page_text = current_user.organization.page_text
```

Instead we want to load the PageText and let the policy check for `pre_check :verify_organization!` check the user's org matches the PageText's org.